src/tx/script/op-codes/op-code.ts

```
@@ -0,0 +1,1175 @@
+        if (vm.dstack.length < 2) TxScriptError.throwInvalidStackOperation(2, vm.dstack.length);
+
+        const newStack = vm.dstack.splice(vm.dstack.length - 2, 2);
+        if (areUint8ArraysEqualSingle(newStack[0], newStack[1])) vm.dstack.push(new Uint8Array([1]));
+        else vm.dstack.push(new Uint8Array());
+
+        break;
+      }
+
+      case OpCodes.OpEqualVerify:
+        if (vm.dstack.length < 2) TxScriptError.throwInvalidStackOperation(2, vm.dstack.length);
+        const newStack = vm.dstack.splice(vm.dstack.length - 2, 2);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)